README.md

```diff
@@ -68,10 +65,10 @@ their mean value for each feature is computed and used in the analysis. See the
 'Theory' section below for a discussion of this approach.
 
 ## API
-The `gibbs` and `gibbs_loo` functions due minimal preprocessing on the input data.
-The source and sink dataframes are treated as final (no collapsing is done on them),
-i.e. each sink is treated independently. The data is **not** rarified, the tables
-are expected to have the desired row sums.
+The `gibbs` function does no preprocessing. Validty checks are done
```

Comment:
How about:

The ``gibbs`` function does not perform collapsing or rarefaction. Validity checks are performed. If `sources`  and `sinks`...